Rocket.Chat/Models/AuthSettings.swift

```diff
@@ -30,6 +30,17 @@ final class AuthSettings: BaseModel {
     @objc dynamic var isGoogleAuthenticationEnabled = false
     @objc dynamic var isLDAPAuthenticationEnabled = false
 
+    // Registration
+    enum RegistrationFormAccess: String {
+        case isPublic = "Public", isDisabled = "Disabled", isSecretURL = "Secret URL"
+    }
+    @objc dynamic var rawRegistrationForm: String?
+    var registrationForm: RegistrationFormAccess {
+        guard let rawValue = rawRegistrationForm,
+            let value = RegistrationFormAccess(rawValue: rawValue) else { return .isPublic }
```

Comment:
Could you change for this?

```swift
guard let rawValue = rawRegistrationForm,
      let value = RegistrationFormAccess(rawValue: rawValue)
else {
     return .isPublic
}

Reply:
Ok for aligning vertically but are you sure about the else on a new line? That's the only change I didn't make 😈

Reply:
@sowenjub Yes, please! 😂 That's how we're doing in other places of the app for guards with multiple lines: https://github.com/RocketChat/Rocket.Chat.iOS/blob/1aba0fcdf2993e108925ca8b39baec79fa829bf0/Rocket.Chat/OAuthManager.swift#L50-L61.

Reply:
Alright, alright, pushed